native/tests/hdr-tone-mapping.test.sh

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# issue #66：libmpv 桥接必须显式下发 HDR tone mapping 配置，使 HDR10/HLG/BT.2020 内容
+# 在目标电视上按 bt.2390 确定性 tone map 到 SDR，而不是依赖 mpv 平台默认猜测。
+set -euo pipefail
+
+readonly ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)